.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 php:
-  - 5.6
-  - 7.0
   - 7.1
```

Comment:
PHP 7.1 is also EOL, should we also remove it from the build matrix and consider it unsupported?

Reply:
PHP 5.6 and 7.0 were requiring implementation of specific code, so removing them permits some code simplification.
For PHP 7.1, I guess it can be removed too.
Let me know if you want me to remove its support.

Reply:
yep, drop it

Reply:
Also, I missed something in my previous review: as PHP8 compat. is on the work here, we should remove it from the allowed failures don't you think?

Reply:
As `nightly` will certainly correspond to PHP 8.1 in a few weeks, I tried, without success, to add a `8.0snapshot` PHP version in the matrix.
So I removed the `allow_failures` directive, but in a near future, when Travis will bundle a stable PHP 8.0 in their environment, it will be necessary to apply this patch:
```diff
diff --git a/.travis.yml b/.travis.yml
index 178957f8..f9da680f 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -4,10 +4,14 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0
   - nightly
 cache:
   directories:
     - vendor
+matrix:
+  allow_failures:
+    - php: nightly
 notifications:
   irc: "irc.freenode.org##atoum"
   webhooks:
```